tests/test_eclmateriallawmanager.cpp

```diff
@@ -236,7 +236,7 @@ int main()
 
     {
         typedef Opm::EclMaterialLawManager<MaterialTraits> MaterialLawManager;
-        typedef typename MaterialLawManager::MaterialLaw MaterialLaw;
+        typedef MaterialLawManager::MaterialLaw MaterialLaw;
```

Comment:
> I'm pretty sure that newer compilers accept this because the final version of C++ 2011 allows this even in non-template contexts.

Thanks, was not aware of that (or forgot about it).